index.html

```diff
@@ -0,0 +1,188 @@
+                </form>
+                <form class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
+                    </div>
+                    <button type="button">See Project</button>
+                </form>
+                <form class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
+                    </div>
+                    <button type="button">See Project</button>
+                </form>
+            </div>
+        </div>
+        <section class="about">
+            <div class="about-text">
+                <div class="about-section">
+                    <h3>About me</h3>
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                        of
+                        my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                        to
+                        contact me.</p>
+                    <button type="button">Get My Resume</button>
+                </div>
+                
+            </div>
+            <div class="language">
+                <ul>
+                    <li>
+                        <h4>Language</h4>
+                        <ul>
+                            <li>Javascript</li>
+                            <li>Ruby</li>
+                            <li>Html</li>
+                            <li>Css</li>
+                        </ul>
+                    </li>
+                    <li>
+                        <h4>Frameworks</h4>
+                        <ul>
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </li>
+                    <li>
+                        <h4>Skills</h4>
+                        <ul>
+                            <li>Codekit</li>
```

Comment:
- [ ] Please, make sure that your design is similar to the design in the Figma template.
Your list of technologies  should look like this:
![image](https://user-images.githubusercontent.com/71874425/153445428-8b9d6fb2-59b4-4596-8d8b-119dbbc5993d.png)